<details><summary>Failed to submit graph for ff19d2f442ba225f027b66e573c409097a024d24
</summary>Message:
Failed to run command: 'pbp generate from-feedstock --output /tmp/tmp9ovpkn85/cargo-bundle-licenses-feedstock.yaml --custom-metadata-file /tmp/tmp9ovpkn85/custom_metadata.yaml --release-name cargo-bundle-licenses-feedstock-3-ff19d2f442ba225f027b66e573c409097a024d24 cargo-bundle-licenses-feedstock/recipe', with error: 'INFO:root:Found feedstock cargo-bundle-licenses-feedstock at /tmp/tmp9ovpkn85/cargo-bundle-licenses-feedstock/recipe
INFO:root:Build/Test graph from cargo-bundle-licenses-feedstock
Created by sbx_user1051 on 2026-03-11 13:29:19
Command line: /opt/conda/envs/webhook-listener/bin/pbp generate from-feedstock --output /tmp/tmp9ovpkn85/cargo-bundle-licenses-feedstock.yaml --custom-metadata-file /tmp/tmp9ovpkn85/custom_metadata.yaml --release-name cargo-bundle-licenses-feedstock-3-ff19d2f442ba225f027b66e573c409097a024d24 cargo-bundle-licenses-feedstock/recipe

INFO:root:Creating graph from feedstock at: cargo-bundle-licenses-feedstock
INFO:pbpcli.utils.common:Setting env var: CONDA_SUBDIR = linux-64.
INFO:pbpcli.utils.expand_variants:Elapsed time for get_package_variants: 0.12 seconds
INFO:pbpcli.utils.expand_variants:Matrix vars: 2 unchanging vars:(488), changing vars (2): ['blas_impl', 'python']
INFO:pbpcli.utils.expand_variants:Elapsed time for preparation for render: 0.00 seconds
INFO:pbpcli.utils.common:Setting env var: CONDA_PKGS_DIRS = /tmp/conda_pkgs_03u6fzr6.
WARNING: Number of parsed outputs does not match detected raw metadata blocks. Identified output block may be wrong! If you are using Jinja conditionals to include or exclude outputs, consider using `skip: true  # [condition]` instead.
INFO:pbpcli.utils.common:Unsetting env var: CONDA_PKGS_DIRS
INFO:pbpcli.utils.common:Unsetting env var: CONDA_SUBDIR
Traceback (most recent call last):
  File "/opt/conda/envs/webhook-listener/bin/pbp", line 10, in <module>
    sys.exit(cli())
             ~~~^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1442, in __call__
    return self.main(*args, **kwargs)
           ~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1363, in main
    rv = self.invoke(ctx)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1830, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1830, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1226, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 794, in invoke
    return callback(*args, **kwargs)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 416, in wrapper
    return func(*args, **kwargs)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 643, in from_feedstock
    build_configs, test_configs = expand_build_matrix(
                                  ~~~~~~~~~~~~~~~~~~~^
        feedstock.recipe_dir_path, src_channels, parsed_user_variants
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 82, in expand_build_matrix
    _targets, build_configs, test_configs = variants_for_feedstock_and_platform(
                                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        recipe_dir_path, platform, src_channels, user_variants
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/utils/expand_variants.py", line 554, in variants_for_feedstock_and_platform
    metadata_tuples = api.render(
        parsed.recipe,
    ...<2 lines>...
        variants=user_variants,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/api.py", line 68, in render
    return render_metadata_tuples(
        metadata_tuples,
    ...<3 lines>...
        bypass_env_check=bypass_env_check,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/render.py", line 1097, in render_metadata_tuples
    for od, om in meta.get_output_metadata_set(
                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        permit_unsatisfiable_variants=permit_unsatisfiable_variants,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        permit_undefined_jinja=not finalize,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        bypass_env_check=bypass_env_check,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ):
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/metadata.py", line 2725, in get_output_metadata_set
    for k in out_metadata.get_used_vars()
             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/metadata.py", line 2915, in get_used_vars
    script_reqs = self._get_used_vars_output_script()
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/metadata.py", line 3038, in _get_used_vars_output_script
    script = os.path.join(self.path, this_output["script"])
  File "<frozen posixpath>", line 90, in join
  File "<frozen genericpath>", line 188, in _check_arg_types
TypeError: join() argument must be str, bytes, or os.PathLike object, not 'list'
'
aws lambda request id: b514df17-d699-58f7-9a9a-bd3ca6f116ce
[View in Grafana](https://grafana.anacondaconnect.com/explore?schemaVersion=1&panes=%7B%2286g%22:%7B%22datasource%22:%22loki-central%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7B__aws_cloudwatch_log_group%3D%5C%22%2Faws%2Flambda%2Fpbp-graph-submitter-prod%5C%22%7D%20%7C%20json%20%7C%20requestId%20%3D%20%60b514df17-d699-58f7-9a9a-bd3ca6f116ce%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22loki-central%22%7D,%22editorMode%22:%22builder%22,%22direction%22:%22backward%22%7D%5D,%22range%22:%7B%22from%22:%22now-3h%22,%22to%22:%22now%22%7D,%22panelsState%22:%7B%22logs%22:%7B%22columns%22:%7B%220%22:%22Time%22,%221%22:%22Line%22%7D,%22visualisationType%22:%22table%22,%22labelFieldName%22:%22labels%22%7D%7D%7D%7D&orgId=1)</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -17,9 +17,8 @@ build:
 
 outputs:
   - name: {{ name }}
-    script:
-      - build-unix.sh  # [not win]
-      - bld-win.bat    # [win]
+    script: build-unix.sh  # [not win]
+    script: bld-win.bat  # [win]
     requirements:
       build:
         - {{ compiler('rust') }}

